src/templates/results.html

```diff
@@ -6,15 +6,19 @@
 <div class="container">
     <div class="row">
         <div class="col-md-12">
-            <h1>Quiz results for user_id 1</h1>
-            {% for attempt in quiz_attempts %}
-                <p>{{ loop.index }}</p>
+            <h1>Quiz results for {{ user_id }}</h1>
+            {% if not user_id %}
+                <p>Add ?userId=yourUserId to the end of this URL to see your results.</p>
```

Comment:
Well snap, I already have that copy. Just need to not pass 1 in the event they don't have that get param.